src/Grpc.Net.Client.Web/Internal/GrpcWebRequestContent.cs

```diff
@@ -61,6 +61,16 @@ private async Task SerializeTextToStreamAsync(Stream stream)
 
         protected override bool TryComputeLength(out long length)
         {
+            // ContentLength is calculated using the inner content's TryComputeLength.
+            var contentLength = _inner.Headers.ContentLength;
+            if (contentLength != null)
+            {
+                length = _mode == GrpcWebMode.GrpcWebText
```

Comment:
Curious, why did the return value change and where did this computation originate?